impeller/aiks/aiks_context.cc

```diff
@@ -36,13 +36,16 @@ const ContentContext& AiksContext::GetContentContext() const {
   return *content_context_;
 }
 
-bool AiksContext::Render(const Picture& picture, RenderTarget& render_target) {
+bool AiksContext::Render(const Picture& picture,
+                         RenderTarget& render_target,
+                         bool wait_until_completed) {
```

Comment:
here and elsewhere, consider using an enum instead of a bool (perhaps just reusing CommandBuffer::SyncMode).

Reply:
Done.